src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/DetailControl.xaml.cs

```diff
@@ -56,14 +60,25 @@ private void PackageSolutionDetailControl_DataContextChanged(object sender, Depe
             }
         }
 
-        private void ExecuteOpenLicenseLink(object sender, ExecutedRoutedEventArgs e)
+        /// <summary>
+        /// Handles Hyperlink controls inside this DetailControl class associated with
+        /// <see cref="PackageManagerControlCommands.OpenExternalLink" />
+        /// </summary>
+        /// <param name="sender">A Hyperlink control</param>
+        /// <param name="e">Command arguments</param>
+        private void ExecuteOpenExternalLink(object sender, ExecutedRoutedEventArgs e)
         {
             var hyperlink = e.OriginalSource as Hyperlink;
-            if (hyperlink != null
-                && hyperlink.NavigateUri != null)
+            if (hyperlink != null && hyperlink.NavigateUri != null)
             {
                 Control.Model.UIController.LaunchExternalLink(hyperlink.NavigateUri);
                 e.Handled = true;
+
+                if (e.Parameter is not null and HyperlinkType hyperlinkType)
+                {
+                    var evt = new HyperlinkClickedTelemetryEvent(hyperlinkType, UIUtility.ToContractsItemFilter(Control.ActiveFilter), Control.Model.IsSolution);
+                    TelemetryActivity.EmitTelemetryEvent(evt);
+                }
```

Comment:
No right now, because not all Hyperlink controls are tracked. You need to add ad CommandParameter. This `if` conditiional guards agains hyperlink controls without that CommandParameter